wire-ios/Wire-iOS/Sources/UserInterface/Settings/Debug Report/LogFilesProvider.swift

```diff
@@ -0,0 +1,104 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import WireCommonComponents
+import WireSystem
+import ZipArchive
```

Comment:
Foundation is not needed here:

```suggestion
import WireCommonComponents
import WireSystem
import ZipArchive
```

Reply:
done https://github.com/wireapp/wire-ios/pull/1658/commits/8c85d9ddbe3d4f96ae06fb91a9db48865e963819